server/src/lib/validation/jobs.test.ts

```diff
@@ -14,7 +10,8 @@ describe('Job Schema Validation', () => {
     };
     const res: any = {};
     const next = jest.fn();
-    jobValidate(req, res, next);
+    const van = validate({ params: jobSchema });
```

Comment:
Question not a request for change, why is it 'van'?

Reply:
ha, just a terrible variable name. I'll update it to something less random.